### PR TITLE
fix: simplify local development

### PR DESCRIPTION
**Pull request by johanandren**
Refs https://github.com/lightbend/akkaserverless-java-sdk/pull/436

Single docker compose file for all

### CODE REVIEW

**Comment by retgits** [comment]
Cool! We'll need to update this in https://developer.lightbend.com/docs/akka-serverless/services/running-locally.html too :)

**Comment by johanandren** [comment]
Thats is done already in https://github.com/lightbend/akkaserverless-docs/pull/948 if I didn't miss anything.

**Comment by retgits** [comment]
Ah great! Totally missed that PR… sorry 😅

File: samples/js/valueentity-counter/.dockerignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 /user-function.desc
-/docker-compose.linux.yml
-/docker-compose.yml
```
Comment (pvlugter) [inline samples/js/valueentity-counter/.dockerignore:3]: the `docker-compose.yml` not intended to be removed (and can't suggest on deleted lines)

**Review by pvlugter — APPROVED** [review]
LGTM.

Just a small correction needed to one of the `.dockerignore` files. I can push directly.

These changes need to be applied to other open PRs with new samples.

**Comment by pvlugter** [comment]
Related to the similar changes in Java SDK: should we be adding a `HOST=0.0.0.0` to the `npm start` or similar?

**Pull request merged**